irods_testing_environment/logs.py

```diff
@@ -23,17 +23,44 @@ def configure(verbosity=1, log_filename=None):
         handlers = handlers
     )
 
-def collect_logs(docker_client, containers, output_directory, logfile_path=None):
-    if not logfile_path:
-        logfile_path = os.path.join(context.irods_home(), 'log')
 
-    for c in containers:
-        od = os.path.join(output_directory, 'logs', c.name)
-        if not os.path.exists(od):
-            os.makedirs(od)
+def log_directory_for_version(version):
+    """Return default iRODS log directory for the given `version`."""
+    major,minor,patch = version
 
-        logging.info('saving log to [{}] [{}]'.format(od, c.name))
+    if int(major) < 4:
+        raise NotImplementedError('nothing prior to iRODS 4.0.0 is supported right now')
 
-        archive.copy_from_container(docker_client.containers.get(c.name),
-                                    logfile_path,
-                                    path_to_destination_directory_on_host=od)
+    if int(major) > 4:
+        raise NotImplementedError('the detected iRODS version does not exist yet')
+
+    if int(minor) < 2:
+        return os.path.join(context.irods_home(), 'iRODS', 'log')
+    elif int(minor) < 3:
+        return os.path.join(context.irods_home(), 'log')
+    elif int(minor) < 4:
+        return os.path.join('/var', 'log', 'irods')
+
+    raise NotImplementedError('the detected iRODS version does not exist yet')
+
+
+def log_directory(container):
+    """Return the default iRODS log directory for the detected iRODS version on `container`."""
+    from . import irods_config
+
+    return log_directory(irods_config.get_irods_version(container))
```

Comment:
log_directory_for_version() i think?

Reply:
Mm, indeed. I ended up not using this particular function but figured it might be helpful in the future. Should we just remove it?

Reply:
torch it, easy to implement later.

Reply:
Done